.github/workflows/test-build-rpm.yml

```diff
@@ -340,8 +340,95 @@ jobs:
           cd $GITHUB_WORKSPACE/../k-NN
           ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest
 
-  Cleaning-Host:
+  Test-Kibana-Plugins-Without-Security:
```

Comment:
Do we have with and without security testing in kibana too?

Reply:
I think so. Different users might see different UIs

Reply:
So do we need to mention it in all job names? Just saw without security here! For all others it is not mentioned whether it is with or without security

Reply:
I like to follow the rule where security enabled is default and we only mention "No-Security" when there is no security.

Reply:
Currently different workflow files use different naming conventions. I will create a backlog issue for cleanup

Reply:
https://github.com/opendistro-for-elasticsearch/opendistro-infra/projects/8